spec/requests/api_v1_work_spec.rb

```diff
@@ -182,6 +183,7 @@
                                            "has_public_files" => false
                                          },
                                          "funder" => nil,
+                                         "funder_project_ref" => nil,
```

Comment:
We have a nil version of this `funder_project_ref`, is there also a working example that checks the data and datatype being returned?